README.adoc

```diff
@@ -160,6 +160,7 @@ To build these microservices, navigate to the `start` directory and run the foll
 
 [role=command]
 ```
+cd start
 mvn package
 ```
```

Comment:
I think we should reword the preceding paragraph as it can be misinterpreted as going into the `start` directory and then running the commands which include another `cd start` command. 

I think an alternate rewording could be:

> Navigate to the `start` directory and build the microservices
> [code block here]

Reply:
I changed it to the following:

> Navigate to the `start` directory and build these microservices by running the following commands:

What do you think?